### DISCUSSION TITLE
Deploy Strapi and VueJS on singe server

### DISCUSSION THREAD

**Discussion by dominikjasek** [strapi/strapi#8392] [Legacy Topics]
Hi,

I develop an application running on Strapi as a backend and vuejs as a fronted. I want to deploy it to heroku to a **single app** because I do not want to purchase paid account on heroku. 

Can you guide me how to do this? Is this possible using **Docker**?

I found [this](https://stackoverflow.com/questions/63118276/how-to-deploy-strapi-backend-and-reactjs-frontend-to-a-single-heroku-app) but I am not too aware how I should follow that.

Thanks

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
